buildwin.bat

```diff
@@ -0,0 +1,13 @@
+
+REM Use the downloaded OpenSSL, for all other libraries the bundled version is used.
+set BORG_OPENSSL_PREFIX=C:\OpenSSL-v111-Win64
```

Comment:
maybe remove the version number from the directory.
guess you don't want to update the script just because the version changed from 1.1.1 to 1.1.2.

see also above, the readme could specify which version / arch is needed.

Reply:
Point is: Appveyor offers different OpenSSL Versions (see https://www.appveyor.com/docs/windows-images-software/#tools), therefore I named the folder the same on my system.

Reply:
OK. Good point, so that just need to be mentioned **above that line**.